Sources/PrettyPrint/TokenStreamCreator.swift

```diff
@@ -482,6 +480,12 @@ private final class TokenStreamCreator: SyntaxVisitor {
   }
 
   override func visit(_ node: WhileStmtSyntax) {
+    before(node.firstToken, tokens: .open(.inconsistent, 6))
```

Comment:
The presence of a label in front of the keyword makes the choice of 6 here kind of arbitrary, but I don't really have a good alternative; using an indentation based on the length of the label would be even worse. So I guess it's fine as is.